GiTinder/Services/UserServices.cs

```diff
@@ -12,6 +27,46 @@ public UserServices(GiTinderContext context)
         {
             _context = context;
         }
+<<<<<<< HEAD
+        //async method is used everytime something should take long time like for requesting for data await allows certain line to run independently.
+        //if user doesn't exist in github then it shouldnt respond
+        public async Task<User> GetUserAsync(string username)
+        {
+    
+            HttpClient client = new HttpClient();
+            //client.DefaultRequestHeaders.Clear() to clear headers being sure that it should only have requested headers. Learning
+            //Gihub API needs to track requests through header User Agent (https://developer.github.com/v3/#user-agent-required).
+            client.DefaultRequestHeaders.Add("User-Agent", "GiTinderApp");
```

Comment:
@Riceqrisp extract these two lines into a separate method

Reply:
Extracted to HeadersSettingForGitHubApi()

Reply:
Moved private HttpClient client = new HttpClient(); to fields.